Project.toml

```diff
@@ -3,14 +3,18 @@ uuid = "787d08f9-d448-5407-9aad-5290dd7ab264"
 version = "0.2.5"
 
 [deps]
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
+ColorVectorSpace = "c3611d14-8923-5661-9e6a-0046d554d3a4"
```

Comment:
`ColorTypes` is reexported by `ImageCore` here, so this can be removed.

Can you add a line for `ColorVectorSpace` in compat section also? `ColorVectorSpace = "0.7, 0.8"` might be a good choice.